docs/DeveloperGuide.adoc

```diff
@@ -723,15 +723,56 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new person |populate the App
 
 |`* * *` |user |delete a person |remove entries that I no longer need
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |import users from an Excel file |easily restore a backup or add new contacts in bulk
+
+|`* * *` |user |see a sorted contact list after sorting |save time from not needing to enter another list command
+
+|`* * *` |user |export all persons in the App out |create a backup of all existing persons in the App
+
+|`* * *`|user |add a person without filling all the fields |add even with missing information
+
+|`* * *`|user |tag my contacts |I can categorise my contacts into groups
+
+|`* * *`|user |edit my contacts |I can update my contacts details
+
+|`* * *`|group member |filter contact list by tags |see only my group mates
+
+|`* *`|user |copy listed emails into my clipboard |convenient when emailing to a batch of contacts
+
+|`* *`|user |add optional fields for contact details |add different means of communication with a particular contact
+
 |`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
 
+|`* *` |new user |have a simple walkthrough |quickly learn how the user interface works
+
+|`* *` |new user |have all the possible commands listed out on first start |I can learn how to use the application immediately
+
+|`* *` |user |be able to add remarks to my contacts |I can add a note if there is something important about the contact
+
+|`* *` |user |choose where to save my file |I can have control over where my files are stored
+
+|`* *` |user |be able to email my contacts |I send an email to my contacts easily
+
+|`* *` |user |add multiple address book |I can have a merge copy of my contacts from different platform
+
+|`* *` |user |add a display photo to an existing person |remember how a particular person looks like
+
+|`* *` |user with many persons in the address book |merge two persons into one |merge duplicated entries of the same person in the list
+
+|`* *` |user |add a new person in any order |so that I can have an easier time adding a person into the App without having to follow a fixed format
```

Comment:
may be re-word this? the `I want to ...` part doesn't really lead to the `so that ...` part